README.md

```diff
@@ -1001,7 +1011,7 @@ console.log(trade);
           '0x1985365e9f78359a9B6AD760e32412f4a445E862',
         ],
         uniswapVersion: 'v3',
-        liquidityProviderFee: 0.0005
+        liquidityProviderFee: [0.003, 0.005, 0.003],
```

Comment:
if your getting a quote as it is now it picks them best price but also factor in the provider fee i would probably expect 3 rows in this case no?